### ISSUE TITLE
invalid value for 'warning.length'

### ISSUE THREAD

**Issue by tpapp** [stan-dev/rstan#173]
I get the following error (with stacktrace). Versions below.

**Update**: The proximate cause is that `R` has a hardcoded limit of `8170` for `warning.length`.  Suggested workaround: truncate the error message accordingly.

``` R
> options(error=recover)
> 
+ + + 
TRANSLATING MODEL 'study-truncation-2' FROM Stan CODE TO C++ CODE NOW.
Error in options(warning.length = error_msg_len) (from stan.R#41) : 
  invalid value for 'warning.length'

Enter a frame number, or 0 to exit   

 1: .ess.eval("fit2 <- stan(\"study-truncation-2.stan\",\n             data=lis
 2: .ess.source(file, echo = echo, print.eval = print.eval, max.deparse.length 
 3: ss(file, echo = echo, local = local, print.eval = print.eval, max.deparse.l
 4: withVisible(eval(ei, envir))
 5: eval(ei, envir)
 6: eval(expr, envir, enclos)
 7: study-truncation.org!1898H9q#1: stan("study-truncation-2.stan", data = list
 8: stan.R#41: suppressMessages(do.call(rstan::stan, args = dots))
 9: withCallingHandlers(expr, message = function(c) invokeRestart("muffleMessag
10: do.call(rstan::stan, args = dots)
11: (function (file, model_name = "anon_model", model_code = "", fit = NA, data
12: stan_model(file, model_name = model_name, model_code = model_code, boost_li
13: stanc(file = file, model_code = model_code, model_name = model_name, verbos
14: options(warning.length = error_msg_len)

Selection: Enter an item from the menu, or 0 to exit
```

R and package versions:

``` R
> R.version
               _                           
platform       x86_64-pc-linux-gnu         
arch           x86_64                      
os             linux-gnu                   
system         x86_64, linux-gnu           
status                                     
major          3                           
minor          2.1                         
year           2015                        
month          06                          
day            18                          
svn rev        68531                       
language       R                           
version.string R version 3.2.1 (2015-06-18)
nickname       World-Famous Astronaut      
> help(package=rstan)

        Information on package ‘rstan’

Description:

Package:       rstan
Type:          Package
Title:         RStan: R interface to Stan
Version:       2.6.0
Date:          2015-02-06
```

**Comment by tpapp** [comment]
I found that this is fixed by 3af5981d64f3296aa5d5bdc739cd1a1a556ac8f2. Sorry for the noise.

**Comment by bob-carpenter** [comment]
Thanks for reporting in the first place.  
I'd much rather err in favor of too many than too few
bug reports.
- Bob
